index.html

```diff
@@ -286,48 +290,50 @@ <h2>Languages</h2>
 							</ul>							
```

Comment:
- [ ] Good job on adding the button interactions as requested by the previous reviewer 👊🏽 However the button in your last work item doesn't have any effects 🤔. Kindly take a look at the screenshot below and try to determine any missing classes from the button then fix the issue 👇🏽:
![image](https://user-images.githubusercontent.com/43172164/200056539-4a647e62-0ec2-4f72-8fa8-b4d3cf268114.png)